keps/2724-topology-aware-scheduling/README.md

```diff
@@ -478,6 +485,14 @@ type PodSetTopologyRequest struct {
   // +optional
   Preferred *string `json:"preferred,omitempty"`
 
+  // Unconstrained indicates the topology assignment for the PodSet should be,
+  // computed using the Unconstrained algorithm. Indicated by the
```

Comment:
IIUC we don't have "Unconstrained algorithm", but in any case, I think we can assume the algorithm is not tightly coupled to the annotation definition.

Reply:
Done